sky/data/mounting_utils.py

```diff
@@ -55,6 +59,67 @@ def get_gcs_mount_cmd(bucket_name: str, mount_path: str) -> str:
     return mount_cmd
 
 
+def get_az_mount_install_cmd() -> str:
+    """Returns a command to install AZ Container mount utility blobfuse2.
+
+    Returns:
+        str: installation command of the mounting utility blobdfuse2
+    """
+    install_cmd = ('sudo apt-get update; '
+                   'sudo apt-get install -y '
+                   '-o Dpkg::Options::="--force-confdef" '
+                   'fuse3 libfuse3-dev && '
+                   'wget -nc https://github.com/Azure/azure-storage-fuse'
+                   f'/releases/download/blobfuse2-{BLOBFUSE2_VERSION}'
+                   f'/blobfuse2-{BLOBFUSE2_VERSION}-Debian-11.0.x86_64.deb '
+                   '-O /tmp/blobfuse2.deb && '
+                   'sudo dpkg --install /tmp/blobfuse2.deb && '
+                   f'mkdir -p {_BLOBFUSE_CACHE_DIR};')
+
+    return install_cmd
+
+
+def get_az_mount_cmd(container_name: str,
+                     storage_account_name: str,
+                     mount_path: str,
+                     storage_account_key: Optional[str] = None) -> str:
+    """Returns a command to mount an AZ Container using blobfuse2.
+
+    Args:
+        container_name: str; name of the mounting container
+        storage_account_name: str; name of the storage account the given
+            container belongs to
+        mount_path: str; path where the container will be mounting
+        storage_account_key: Optional[str]; access key for the given storage
+            account
+
+    Returns:
+        str: command used to mount AZ container with blobfuse2
+    """
+    # Note: Using MOUNT mode with Azure Blob Storage by default gives full
+    # permissions(777) to files/directories unlike other object
+    # storages(s3, gcs). It is also impossible to change these
+    # permissions(chmod). Special type of storage account is required to
```

Comment:
```suggestion
    # Note: Using MOUNT mode with Azure Blob Storage by default gives full
    # permissions (777) to files/directories unlike other object
    # storages (S3, GCS). It is also impossible to change these
    # permissions (chmod). Special type of storage account is required to
```
nit

Reply:
updated

Reply:
This comment was removed when we fixed the file permissions to `755`.